configure

```diff
@@ -100,6 +102,32 @@ setup_git() {
   cd - >/dev/null 2>&1
 }
 
+setup_go() {
+  local user_override="${1}"; shift
+
+  if [ -n "${user_override}" ]; then
+    GOROOT="${user_override}"
+  else
+    local go="$(find_progs go)"
+    [ -n "${go}" ] && GOROOT="$(dirname "$(dirname "${go}")")"
+  fi
+
+  if [ -z "${GOROOT}" ]; then
+    info "Go not found; cannot run integration tests"
+  else
+    info "Using Go with GOROOT: ${GOROOT}"
+
+    mkdir -p .gopath/src/github.com/bazelbuild
+    rm -f .gopath/src/github.com/bazelbuild/sandboxfs
+    ln -s ../../../.. .gopath/src/github.com/bazelbuild/sandboxfs
+
+    for dep in bazil.org/fuse golang.org/x/net/context golang.org/x/sys/unix; do
```

Comment:
Strongly recommend moving to Go modules soon. :-)

Reply:
I'll have to read about that then. Haven't followed new developments much!